os/kernel/Kconfig

```diff
@@ -1081,6 +1081,9 @@ config MPU_STACKGAURD
 	bool "protect thread stack"
 	default n
 
+config MPU_STACKGAURD
```

Comment:
what is this?

Reply:
sorry, didn't checked it, its duplicate. will remove it.